packages/shared/src/components/squads/SquadHeaderMenu.tsx

```diff
@@ -59,34 +69,24 @@ export default function SquadHeaderMenu({
       animation="fade"
     >
       {isSquadOwner && (
-        <Item className="typo-callout" onClick={onEditSquad}>
-          <span className="flex items-center w-full typo-callout">
-            <EditIcon size="medium" secondary={false} className="mr-2" /> Edit
-            Squad details
-          </span>
-        </Item>
+        <ContextMenuItem
+          Icon={EditIcon}
+          onClick={onEditSquad}
+          label="Edit Squad details"
+        />
       )}
       {/* <Item className="typo-callout"> */}
       {/*  <span className="flex items-center w-full typo-callout"> */}
       {/*    <TourIcon size="medium" secondary={false} className="mr-2" /> Learn */}
       {/*    how Squads work */}
       {/*  </span> */}
       {/* </Item> */}
-      {isSquadOwner ? (
-        <Item className="typo-callout" onClick={onDeleteSquad}>
-          <span className="flex items-center w-full typo-callout">
-            <TrashIcon size="medium" secondary={false} className="mr-2" />{' '}
-            Delete Squad
-          </span>
-        </Item>
-      ) : (
-        <Item className="typo-callout" onClick={onLeaveSquad}>
-          <span className="flex items-center w-full typo-callout">
-            <ExitIcon size="medium" secondary={false} className="mr-2" /> Leave
-            Squad
-          </span>
-        </Item>
-      )}
+      <ContextMenuItem
```

Comment:
this should be hidden on non-mobile i think

Reply:
Agreed.